services/movie_session.py

```diff
@@ -42,3 +42,9 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    return list(MovieSession.objects.filter(
+        id=movie_session_id).values("cinema_hall__rows",
+                                    "cinema_hall__seats_in_row"))
```

Comment:
The `get_taken_seats` function does not accurately retrieve the taken seats for a movie session. Consider querying the `Ticket` model to get the actual taken seats for the given `movie_session_id`.